sklearn/tree/_export.py

```diff
@@ -714,6 +723,28 @@ def recurse(self, node, tree, ax, max_x, max_y, depth=0):
                     (max_y - node.parent.y - 0.5) / max_y,
                 )
                 ax.annotate(node.tree.label, xy_parent, xy, **kwargs)
+
+                # Draw True/False labels if parent is root node
+                if node.parent.parent is None:
+                    # Adjust the position for the text to be slightly above the arrow
+                    text_pos = (
+                        (xy_parent[0] + xy[0]) / 2 + 0.05,
+                        (xy_parent[1] + xy[1]) / 2,
+                    )
+                    # Annotate the arrow with the edge label to indicate the child
+                    # where the sample-split condition is satisfied
+                    if node.parent.left() == node:
+                        ax.annotate(
+                            "True",
+                            text_pos,
+                            **non_box_kwargs,
+                        )
+                    else:
+                        ax.annotate(
+                            "False",
+                            text_pos,
+                            **non_box_kwargs,
+                        )
```

Comment:
```suggestion
                    ax.annotate(
                        node.parent.left() == node, text_pos, **non_box_kwargs,
                    )
```

This may simplify the code a bit by avoiding the branching. `ax.annotate` will convert boolean into string I believe, but you can try and check.